jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -55,6 +56,11 @@ public class Dispatcher implements RequestDispatcher
      * Dispatch include attribute names
      */
     public static final String __FORWARD_PREFIX = "jakarta.servlet.forward.";
+    
+    /**
+     * Name of original request attribute
+     */ 
+    public static final String __ORIGINAL_REQUEST = "org.eclipse.jetty.originalRequest";
```

Comment:
maybe put ee10 in the name to protect against future cross context dispatches between ee10, ee11 and then back to ee10.